.github/workflows/docker.yaml

```diff
@@ -18,6 +18,47 @@ env:
   PUSH: ${{ (github.event_name != 'pull_request') && (github.repository == 'Robotic-Decision-Making-Lab/blue') }}
 
 jobs:
+  mavros:
```

Comment:
Do we need to host this image if we aren't going to use it as the base image for the rest of the Dockerfile?

Reply:
I do think the mavros build image needs to be built and cached ... somewhere.  And I don't _think_ all of those built layers are cached in `ci` ... it only caches the layer which includes the copied files?

But there's still some poking to ensure that `mavros` cache is loaded by the `ci` layers.